README.md

```diff
@@ -57,9 +57,10 @@ Running in Docker
 
 Running in Kubernetes
 - `./setup.sh` - To run Polaris as a mini-deployment locally. This will create two pods that bind themselves to port `8181`.
-- `kubectl get pods` - To check the status of the pods.
-- `kubectl get deployment` - To check the status of the deployment.
-- `kubectl describe deployment polaris-deployment` - To troubleshoot if things aren't working as expected.
+- `kubectl get pods -n polaris` - To check the status of the pods.
+- `kubectl get deployment -n polaris` - To check the status of the deployment.
+- `kubectl describe deployment polaris-deployment -n polaris` - To troubleshoot if things aren't working as expected.
+- `kubectl port-forward svc/polaris-service -n polaris 8181:8181` - To create a secure connection between a local machine and a pod within the cluster.
```

Comment:
I think we should move this line right after line 59, as it is a part of k8s local deployment, or even better, if we can move it inside setup.sh?